packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -170,6 +170,8 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
 
     // Approximation but it's pretty accurate. Could consider using a layout builder
     // or maybe build in an overlay (that's isn't visible) to compute.
+    // TODO(https://github.com/flutter/devtools/issues/3075) This approximation
+    // doesn't work for the new file opener, should compute the actual height.
     final tileEntryHeight = box.size.height;
```

Comment:
Instead of doing the work to compute the height of the tiles, can you just make the tileEntryHeight configurable by a parameter on the `AutoComplete` widget instead?

Reply:
Good idea, done!